src/main/java/lotto/model/LottoNumber.java

```diff
@@ -0,0 +1,26 @@
+package lotto.model;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoNumber {
+
+    private List<Integer> numberlist;
```

Comment:
인스턴스 변수명에서 세부적인 구현 내용이 안 드러나는게 좋을 것 같습니다:)
```suggestion
    private List<Integer> numbers;
```